test/e2e/run.sh

```diff
@@ -32,6 +32,8 @@ main() {
   else
     run_e2e_tests "$@"
   fi
+  touch /tmp/done
+  sleep infinity
 }
 
-main "$@"
+main "$@" | tee "$RESULT_FILE"
```

Comment:
Would it be better to have the `tee` command living within the `run_e2e-tests` and `run_chaos` functions such that it closes the`RESULT_FILE` properly? Wouldn't we kinda be leaving an open file handle?

Reply:
Yes, you're right, we can move the `tee` and avoid that, though I don't think it can cause an issue.

Reply:
I will take the opportunity to do it only for the e2e tests job, not the chaos job.